pallets/msa/src/rpc/src/lib.rs

```diff
@@ -56,6 +58,13 @@ pub trait MsaApi<BlockHash, AccountId> {
 		provider_msa_id: ProviderId,
 	) -> RpcResult<Option<Vec<SchemaGrant<SchemaId, BlockNumber>>>>;
 
+	/// Retrieve the list of all delegations for a MsaId
+	#[method(name = "msa_getAllGrantedDelegationsByMsaId")]
+	fn get_all_granted_delegations_by_msa_id(
+		&self,
+		delegator_msa_id: DelegatorId,
+	) -> RpcResult<Vec<DelegationResponse<SchemaId, BlockNumber>>>;
+
```

Comment:
nit: do we want to take a an optional blocknumber as input in order to check delegations at specific block(s)

Reply:
I'd say no. If you care about blocks, then you can interact directly with chain state.